### ISSUE TITLE
Strange duplication of directory names

### ISSUE THREAD

**Issue by Simon818** [lightpohl/podcast-dl#47]
This may be more of a "what am I doing wrong" issue so I sincerely apologize if that's the case.

I recently moved from WSL to Raspberry Pi OS and resurrected a script that downloads podcasts to specific directories based on a short name. Here's an example command as output by the script:

`npx podcast-dl --out-dir "/hd/podcasts/talkshow" --archive "/hd/podcasts/talkshow/archive.pdl" --threads 4 --include-meta --include-episode-meta --url https://daringfireball.net/thetalkshow/rss`

This should download the episode and all associated files to /hd/podcasts/talkshow (as far as I know). And on WSL, it did. When I migrated the script to Linux, it started downloading to /hd/podcasts/talkshow/hd_podcasts_talkshow, and the archive (in the main talkshow directory) is called hd_podcasts_talkshow.pdl. this hd_podcasts_talkshow directory is basically the entire path with each / replaced with _.

I think I can just have the script cd into each directory and not specify a path at all, but I'm wondering what I did wrong. The exact same script worked on WSL with a version of podcast-dl from last year; the only thing I changed was the path (from /mnt/e/podcasts to /hd/podcasts).

**Comment by lightpohl** [comment]
Hey! Thanks for opening an issue.

I accidentally introduced a bug in the v8.0.0, which should now be fixed in [v8.0.4](https://github.com/lightpohl/podcast-dl/releases/tag/v8.0.4). If you run `npx pocast-dl@latest`, or update the globally installed version (`npm i -g podcast-dl@latest`), then your script should be back to working as expected.

Sorry for the trouble, and definitely feel free to re-open the issue if you're still having issues!

**Comment by Simon818** [comment]
Thanks for such a quick response and I'm glad it wasn't just me. I was running 8.0.3. I also refactored my script a little so this shouldn't happen regardless, but I'll let you know if I run into anything along these lines.

Also, thanks for making this project at all; I spent a long time trying to find a downloader like this. Most of what I found was missing at least one key feature I needed. Someone on r/datahoarder recommended this and I'm very happy with it.
